test/unit/carriers/shipwire_test.rb

```diff
@@ -24,67 +24,65 @@ def test_response_with_no_rates_is_unsuccessful
 
   def test_successfully_get_international_rates
     date = DateTime.new(2011, 8, 1)
-    DateTime.expects(:now).returns(date).at_least_once
     @carrier.expects(:ssl_post).returns(xml_fixture('shipwire/international_rates_response'))
 
-    response = @carrier.find_rates(
-                 location_fixtures[:ottawa],
-                 location_fixtures[:london],
-                 package_fixtures.values_at(:book, :wii),
-                 :order_id => '#1000',
-                 :items => @items
-               )
-
-    assert response.success?
-
-    assert_equal 1, response.rates.size
-
-    assert international = response.rates.first
-    assert_equal "INTL", international.service_code
-    assert_equal "UPS", international.carrier
-    assert_equal "UPS Standard", international.service_name
-    assert_equal 2806, international.total_price
-    assert_equal date + 7, international.delivery_date
-    assert_equal [date + 1, date + 7], international.delivery_range
+    Timecop.freeze(date) do
+      response = @carrier.find_rates(
+                   location_fixtures[:ottawa],
+                   location_fixtures[:london],
+                   package_fixtures.values_at(:book, :wii),
+                   :order_id => '#1000',
+                   :items => @items
+                 )
+
+      assert response.success?
+
+      assert_equal 1, response.rates.size
+
+      assert international = response.rates.first
+      assert_equal "INTL", international.service_code
+      assert_equal "UPS", international.carrier
+      assert_equal "UPS Standard", international.service_name
+      assert_equal 2806, international.total_price
+      assert_equal [date + 1.day, date + 7.days], international.delivery_range
```

Comment:
Interesting that this doesn't work in all cases, since `date + n` where `n` is an integer and `date` is a `DateTime` should result in adding one day.

Either way, explicitness is good here.